engine/src/test/java/org/camunda/bpm/engine/test/api/task/TaskQueryTest.java

```diff
@@ -32,11 +32,7 @@
 import static org.camunda.bpm.engine.test.api.runtime.TestOrderingUtil.taskByPriority;
 import static org.camunda.bpm.engine.test.api.runtime.TestOrderingUtil.taskByProcessInstanceId;
 import static org.camunda.bpm.engine.test.api.runtime.TestOrderingUtil.verifySortingAndCount;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
Please avoid wildcard imports if you can.

Reply:
fixed